src/components/ChallengeTakePicturesCard.vue

```diff
@@ -19,6 +19,7 @@
         color="primary"
         variant="flat"
         :block="!$vuetify.display.smAndUp"
+        prepend-icon="mdi-image-plus"
         to="/proofs/add/multiple"
       >
         {{ $t('Challenge.StepTakePictures.AddPictures') }}
```

Comment:
reminds me that when you demoed this and I saw "Pictures", I think we should replace elsewhere in the "Images" by "Pictures" (right now we have 3 wordings for the same thing : Proof, Image & Picture I believe)